### PR TITLE
Master ->l10n_il

### PR DESCRIPTION
**Pull request by hedshefer**
Description of the issue/feature this PR addresses:

added vat only and vat self invoice that will fit the israely market needs. In addition, removed unused whd taxes and added employees and customers whd account tags
@jco-odoo 
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by hedshefer** [comment]
@jco-odoo 
The version is ready to be merged to V 15, let me know what do you think.

File: addons/l10n_il/data/account.group.template.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+id,code_prefix_start,code_prefix_end,name,chart_template_id/id
+il_group_100100,100100,100499,"רכוש קבוע",l10n_il.il_chart_template
```
Comment (jco-odoo) [inline addons/l10n_il/data/account.group.template.csv:2]: Should be in English and then translated to Hebrew

File: addons/l10n_il/data/fiscal_templates_data.xml [hunk 2]
```diff
@@ -29,19 +28,70 @@
...
     </record>
 
-    <!-- Export -->
-    <record id="account_fiscal_position_export" model="account.fiscal.position.template">
+    <!-- Import / Export -->
+    <record id="account_fiscal_position_import_export" model="account.fiscal.position.template">
         <field name="name">Import / Export</field>
         <field name="auto_apply" eval="True"/>
         <field name="chart_template_id" ref="il_chart_template"/>
-        <field name="auto_apply" eval="True"/>
```
Comment (jco-odoo) [inline addons/l10n_il/data/fiscal_templates_data.xml:37]: Why do you remove the auto_apply?

File: addons/l10n_il/__manifest__.py [hunk 3]
```diff
@@ -12,14 +12,15 @@
 This module consists of:
  - Generic Israel Chart of Accounts
  - Taxes and tax report
- - Fiscal position for Palestina
+ - Multipule Fiscal positions
```
Comment (jco-odoo) [inline addons/l10n_il/__manifest__.py:15]: Multiple

**Review by jco-odoo — CHANGES_REQUESTED** [review]
please check and squash

File: addons/l10n_il/data/account.group.template.csv (hunk 1 above)
Comment (mart-e) [reply addons/l10n_il/data/account.group.template.csv:2]: @jco-odoo really? I thought it was ok to add untranslated items in localisation modules

Comment (jco-odoo) [reply addons/l10n_il/data/account.group.template.csv:2]: @mart-e But this one is multilang and the CoA is in English

File: addons/l10n_il/data/fiscal_templates_data.xml (hunk 2 above)
Comment (hedshefer) [reply addons/l10n_il/data/fiscal_templates_data.xml:37]: The auto_apply was duplicated

File: addons/l10n_il/__manifest__.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Israel - Accounting',
-    'version': '1.0',
+    'version': '15.0.0',
```
Comment (jco-odoo) [inline addons/l10n_il/__manifest__.py:6]: ?

Comment (jco-odoo) [reply addons/l10n_il/__manifest__.py:6]: module version != odoo version

**Pull request closed without merging**